narwhals/_dask/namespace.py

```diff
@@ -43,6 +50,10 @@ def col(self, *column_names: str) -> DaskExpr:
             backend_version=self._backend_version,
         )
 
+    def all_horizontal(self, *exprs: IntoDaskExpr) -> DaskExpr:
+        # coverage shows 55->exit as uncovered?
```

Comment:
It should apply when conditions are comma separated, otherwise the "&" is resolved by the expr. Meaning:

```py
.filter(nw.col("a") & nw.col("b") #-> enters __and__ method
.filter(nw.col("a"), nw.col("b") #-> enters all_horizontal function to resolve it with __and__ inside it
```

Reply:
thanks! any objections to what I've added, or shall we ship it?

Reply:
I didn't go through the code in its entirety. I agree that  Dask won't allow us to achieve something, so it's ok to move forward and eventually try to address corner cases

Reply:
cool, thanks!